kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/facade/CompanyFacade.java

```diff
@@ -0,0 +1,66 @@
+package com.kodilla.hibernate.manytomany.facade;
+
+import com.kodilla.hibernate.manytomany.Company;
+import com.kodilla.hibernate.manytomany.Employee;
+import com.kodilla.hibernate.manytomany.dao.CompanyDao;
+import com.kodilla.hibernate.manytomany.dao.EmployeeDao;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CompanyFacade {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CompanyFacade.class);
+
+    public List<Company> listOfCompanies(CompanyDao companyDao, String threeChars)
+            throws CompanySearchingException {
+        boolean wasError = false;
+        List<Company> companies =
+                companyDao.retrieveCompanyWithTreeChars(threeChars);
+        try {
+            LOGGER.info("Searching with phrase: " + threeChars);
+            if (companies.size() < 0) {
```

Comment:
To nie jest możliwe, żeby lista miała długość mniejszą niż 0. Ten kod nigdy się nie wykona. Poza tym, dlaczego traktujesz brak znalezionych firm / użytkowników jako błąd? Każda wyszukiwarka przewiduje, że możesz podać argumenty, dla których nie da się nic znaleźć - to jest normalne :)